dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorResource.java

```diff
@@ -0,0 +1,343 @@
+package com.dotcms.rest.api.v1.system.monitor;
+
+import java.io.File;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.dotcms.enterprise.cluster.ClusterFactory;
+import com.dotcms.util.HttpRequestDataUtil;
+import com.dotcms.util.network.IPUtils;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.util.*;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.index.query.QueryBuilders;
+import com.dotcms.repackage.javax.ws.rs.*;
+import com.dotcms.content.elasticsearch.business.IndiciesAPI.IndiciesInfo;
+import com.dotcms.content.elasticsearch.util.ESClient;
+import com.dotcms.repackage.javax.ws.rs.core.Context;
+import com.dotcms.repackage.javax.ws.rs.core.MediaType;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.repackage.javax.ws.rs.core.Response.ResponseBuilder;
+import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.json.JSONObject;
+
+import net.jodah.failsafe.CircuitBreaker;
+import net.jodah.failsafe.Failsafe;
+
+/**
+ * 
+ * 
+ * Call
+ *
+ */
+@Path("/v1/system-status")
+public class MonitorResource {
+
+    static private long LOCAL_FS_TIMEOUT=1000;
+    static private long CACHE_TIMEOUT=1000;
+    static private long ASSET_FS_TIMEOUT=1000;
+    static private long INDEX_TIMEOUT=1000;
+    static private long DB_TIMEOUT=1000;
+    private ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
We already have our own Executor called DotSubmiter, however the Executors won't be never part of a class (as an attribute) since the GC can collect or shutdown the executor and make it invalid, we have already seen this behavior.